hudi-common/src/main/java/org/apache/hudi/common/util/CommitUtils.java

```diff
@@ -49,7 +50,7 @@
 public class CommitUtils {
 
   private static final Logger LOG = LoggerFactory.getLogger(CommitUtils.class);
-  private static final String NULL_SCHEMA_STR = Schema.create(Schema.Type.NULL).toString();
+  private static final String NULL_SCHEMA_STR = HoodieSchema.create(HoodieSchemaType.NULL).toString();
```

Comment:
`HoodieSchema.create(HoodieSchemaType.NULL)` seems to be used in quite a few places, possible to just make this a public constant somewhere where we can just import this to the other relevant classes?